src/views/forms/AccountManagementForms/DepositForm/DepositButtonAndReceipt.tsx

```diff
@@ -221,6 +233,14 @@ export const DepositButtonAndReceipt = ({
     >
       {!canAccountTrade ? (
         <OnboardingTriggerButton size={ButtonSize.Base} />
+      ) : !isConnectedWagmi ? (
+        <Button
+          action={ButtonAction.Primary}
+          onClick={connectWallet}
+          state={{ isLoading: !hasConnectWalletError }}
+        >
+          {stringGetter({ key: STRING_KEYS.CHECK_WALLET_FOR_REQUEST })}
```

Comment:
This is a pretty long string :P

Reply:
hehe do u have thoughts on what to show (Check wallet? Requesting...?) i didn't want to just show the loading dots coz people should be able to click on it again if they want

Reply:
haha might need design input for this? `View Wallet`?

Reply:
updated to show an alert message instead!